_notices/rsn0042.md

```diff
@@ -0,0 +1,60 @@
+---
+layout: notice
+parent: RAPIDS Support Notices
+grand_parent: RAPIDS Notices
+nav_exclude: true
+notice_type: rsn
+# Update meta-data for notice
+notice_id: 42 # should match notice number
+notice_pin: true # set to true to pin to notice page
+
+title: "Moving older RAPIDS conda packages to a `legacy` label"
+notice_author: RAPIDS Ops
+notice_status: Proposal
+notice_status_color: blue
+# 'notice_status' and 'notice_status_color' combinations:
+#   "Proposal" - "blue"
+#   "Completed" - "green"
+#   "Review" - "purple"
+#   "In Progress" - "yellow"
+#   "Closed" - "red"
+notice_topic: Platform Support Change
+notice_rapids_version: "<=v23.02"
+notice_created: 2025-01-27
+# 'notice_updated' should match 'notice_created' until an update is made
+notice_updated: 2025-01-27
+---
+
+## Overview
+
+RAPIDS will move certain older conda packages in the `rapidsai` conda channel
+under a new label: `legacy`. These packages will still be available by appending
+`--label legacy` to installation and search commands.

```

Comment:
I'm a little unclear on this. It looks like `--label` is used when uploading, but for users to install, they need something like `--channel rapidsai --channel rapidsai/label/legacy`. Does that sound correct?

See https://docs.anaconda.com/anacondaorg/user-guide/work-with-labels/#testing-the-discoverability-of-the-package

Reply:
Yup, thanks for catching that @bdice !